example-app/src/main/java/backtraceio/backtraceio/MainActivity.java

```diff
@@ -43,6 +45,7 @@
     @Override
     protected void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
+        BacktraceLogger.setLevel(LogLevel.ERROR);
```

Comment:
I don't think we need this in the demo activity

Reply:
Do we want to set the `LogLevel.ERROR` here? I think for first-time users of our project it could be useful to see the most verbose Backtrace logs by default.